### PR TITLE
Rwahs 35/acquisition data

### PR DESCRIPTION
**Pull request by kehh**
- Contains #1 and that should be merged first.
- Requires https://github.com/rwahs/installation-profile/pull/4
- Import mapping matching the adminDonor data category in https://docs.google.com/spreadsheets/d/11ynTouvmgYI-RoesX3rbvi4xUAhiwTCsOcoUwGLqVHc/edit?usp=sharing
- Add test data for adminDonor data category
- https://youtrack.gaiaresources.com.au/issue/RWAHS-35

### CODE REVIEW

**Comment by leftclickben** [comment]
Assigning so you can resolve the conflicts :-)

**Comment by kehh** [comment]
not too bad actually

**Comment by leftclickben** [comment]
I'm going to leave this for tomorrow.

File: museum/refineryParameters/ItemType.ca_objects.itemType.listItemSplitter.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "list": "itemTypes",
```
Comment (leftclickben) [inline museum/refineryParameters/ItemType.ca_objects.itemType.listItemSplitter.json]: Change to `ItemTypes`

File: museum/sampleRecords/adminDonor.destination.json [hunk 2]
```diff
@@ -0,0 +1,1025 @@
...
+      "ca_objects.AccessionPrefix":"A",
+      "ca_objects.AccessionYear":"1964",
+      "ca_objects.AccessionNumber":"26",
+      "ca_objects.AccessionSeries":"0",
+      "ca_objects.AccessionParts":"a",
+      "ca_objects.AccessionBy":"jm",
+      "ca_entities":"UNKNOWN",
+      "ca_entities.entity_id":"1",
+      "ca_entities.preferred_labels.surname":"UNKNOWN",
+      "ca_entities.Phone":";;;;;;",
```
Comment (leftclickben) [inline museum/sampleRecords/adminDonor.destination.json:78]: Some of these fields have funky, incorrect looking values that are not coming from the source data.

File: museum/sampleRecords/adminDonor.destination.json [hunk 3]
```diff
@@ -0,0 +1,1025 @@
+{
+  "ok":true,
+  "results":[
+    {
+      "object_id":"1",
+      "id":"1",
+      "idno":"A1988.135",
+      "display_label":"STRAWBERRY HILL FARM, ALBANY",
+      "ca_objects.AccessionPrefix":"A",
```
Comment (leftclickben) [inline museum/sampleRecords/adminDonor.destination.json:9]: I don't think these fields (`AccessionPrefix`, `AccessionYear`, `AccessionNumber`) are required as they are components of the `idno` field so they represent a denormalisation (duplicate data).

File: museum/sampleRecords/adminDonor.destination.json [hunk 4]
```diff
@@ -0,0 +1,1025 @@
...
+      "ca_entities.MobilePhone":";;;;;;",
+      "ca_entities.Fax":";;;;;;",
+      "ca_entities.Email":";;;;;;",
+      "ca_entities.DonorNotes":";;;;;;",
+      "ca_entities.Deceased":";;;;;;",
+      "ca_entities.DonorAgentName":";;;;;;",
+      "ca_entities.DonorAgentAddress":";;;;;;",
+      "ca_entities.DonorAgentPhone":";;;;;;",
+      "ca_entities.LastEditBy":"urbaj",
+      "ca_entities.LastEditDate":"March 6 2015",
```
Comment (leftclickben) [inline museum/sampleRecords/adminDonor.destination.json]: Strange date format, should be ISO format, `YYYY-MM-DD` or AU format, `DD MMMM YYYY`.

File: museum/sampleRecords/adminDonor.destination.json (hunk 2 above)
Comment (leftclickben) [inline museum/sampleRecords/adminDonor.destination.json:78]: ...and some of the records have a `SourceName` value of `<<N/A>>` and don't have these fields at all in the destination data.

File: museum/sampleRecords/adminDonor.destination.json (hunk 3 above)
Comment (kehh) [inline museum/sampleRecords/adminDonor.destination.json:9]: I agree but we have an outstanding question on accession number so I've got them in in the interim.

File: museum/sampleRecords/adminDonor.destination.json (hunk 2 above)
Comment (kehh) [inline museum/sampleRecords/adminDonor.destination.json:78]: https://youtrack.gaiaresources.com.au/issue/RWAHS-72 and https://youtrack.gaiaresources.com.au/issue/RWAHS-73

File: museum/sampleRecords/adminDonor.destination.json (hunk 4 above)
Comment (kehh) [inline museum/sampleRecords/adminDonor.destination.json]: This is fixed by ad457eeddb49899e03ffeb4a25304eb94b49294a

**Pull request merged**